src/test/ui/proc-macro/auxiliary/generate-mod.rs

```diff
@@ -1,6 +1,7 @@
 // run-pass
```

Comment:
Pass annotations in `auxiliary` files?
What the hell.
What do they even mean?
`compiletest` should catch them and report an error.

Reply:
I have... no idea :)